octobot_trading/exchanges/exchange_config_data.py

```diff
@@ -53,17 +57,24 @@ def set_config_time_frame(self):  # TODO
     def get_shortest_time_frame(self):
         return self.traded_time_frames[-1]
 
-    async def handle_symbol_update(self, exchange: str, exchange_id: str, crypto_currency: str, symbols: list) -> tuple:
+    async def add_watched_symbols(self, symbols):
+        new_valid_symbols = [
+            symbol
+            for symbol in symbols
+            if self._is_valid_symbol(symbol, []) and symbol not in self.all_config_symbol_pairs
+        ]
         try:
-            return self._add_tradable_symbols(crypto_currency, symbols)
-        except Exception as e:
-            self._logger.exception(e, True, f"Fail to handle symbol update : {e}")
+            if new_valid_symbols:
+                self.all_config_symbol_pairs += new_valid_symbols
```

Comment:
should we really update this list ? It's used to initialize evaluators etc early on, I'm not sure we should update it

Reply:
Ok, I will change it